test/api/v3/integration/groups/POST-groups_groupId_join.js

```diff
@@ -0,0 +1,84 @@
+import {
+  generateUser,
+} from '../../../../helpers/api-integration.helper';
+
+describe('POST /group/:groupId/join', () => {
+  let user;
+  let group;
+  let groupName;
+  let groupType;
+
+  context('Accepting invitation to a guild', () => {
+    beforeEach(async () => {
+      groupName  = 'Test Guild';
+      groupType = 'guild';
+    });
+
+    it('does not give basilist quest to inviter when joining a guild', async () => {
+      user = await generateUser({balance: 1})
+      group = await user.post('/groups', {
+        name: groupName,

```

Comment:
I don't think these need to be variables. There's no need to assert on them later. Why not just:

```
{
  name: 'Test Guild',
  type: 'guild',
}
```